src/display.c

```diff
@@ -337,6 +338,9 @@ static size_t LCUIDisplay_RenderSurface(SurfaceRecord record)
 		for (i = 0; i < (int)rects.length; ++i) {
 			count += LCUIDisplay_RenderSurfaceRect(record,
 							       rect_array[i]);
+#ifdef USE_OPENMP
+			LCUI_FpsMeter_RenderThreadCount(omp_get_num_threads());
```

Comment:
Other modules should not call FpsMeter functions, Because the low-level function code should not depend on the upper-level function code